srcs/raycasting/textures.c

```diff
@@ -0,0 +1,63 @@
+#include "cub.h"
+
+t_dir_code	get_wall_dir(t_cub *cub)
+{
+	if (cub->side == 1 && cub->map_pos->y < cub->pos->y)
```

Comment:
I think we should make an enum for the side as well, that's actually something I had thought about already, but it wasn't that important cos it was only being used in a specific place. But now that it's being used in a completely different place, I think it's really important. Just like this I have no idea if the side 1 is x or y. So we could just do an enum with SIDE_X and SIDE_Y to be 0 and 1

Reply:
Done.